terraform/workspace-variables/app_config.yml

```diff
@@ -48,9 +48,15 @@ development:
 # ey-recovery-content
 content:
   <<: *defaults
+  # jobs
   DASHBOARD_UPDATE: false
+  # cms
   CONTENTFUL_PREVIEW: true
-  # CONTENTFUL_ENVIRONMENT: test
+  CONTENTFUL_ENVIRONMENT: test
```

Comment:
Is this intentional?

Reply:
I'll want to revert this and the timeout envs before merging